src/lib/apiTypesV2/Registration.h

```diff
@@ -25,74 +25,79 @@
 *
 * Author: Fermín Galán Márquez
 */
-
-
 #include <string>
 #include <vector>
 
 #include "apiTypesV2/EntID.h"
 
-/*
 
+#if 0
+------------------------------------------------------------------------------------
   FIXME: Reference registration document (delete once this get implemented)
 
-      {
-            "id": "abcdefg",
-            "description": "Example Context Source",
-            "dataProvided": {
-              "entities": [
-                {
-                  "id": "Bcn_Welt",
-                  "type": "Room"
-                }
-              ],
-              "attrs": [
-                "temperature"
-              ]
-            },
-            "provider": {
-              "http": {
-                "url": "http://contextsource.example.org"
-              },
-              "supportedForwardingMode": "all",
-              "legacyForwarding": true
-            },
-            "expires": "2017-10-31T12:00:00",
-            "status": "failed",
-            "forwardingInformation": {
-              "timesSent": 12,
-              "lastForwarding": "2017-10-06T16:00:00.00Z",
-              "lastFailure": "2017-10-06T16:00:00.00Z",
-              "lastSuccess": "2017-10-05T18:25:00.00Z",
-            }
-}
-
-*/
+  {
+    "id": "abcdefg",
+    "description": "Example Context Source",
+    "dataProvided": {
+      "entities": [
+        {
+          "id": "Bcn_Welt",
+          "type": "Room"
+        }
+      ],
+      "attrs": [
+        "temperature"
+      ]
+    },
+    "provider": {
+      "http": {
+        "url": "http://contextsource.example.org"
+      },
+      "supportedForwardingMode": "all",
+      "legacyForwarding": true
+    },
+    "expires": "2017-10-31T12:00:00",
+    "status": "failed",
+    "forwardingInformation": {
+      "timesSent": 12,
+      "lastForwarding": "2017-10-06T16:00:00.00Z",
+      "lastFailure": "2017-10-06T16:00:00.00Z",
+      "lastSuccess": "2017-10-05T18:25:00.00Z",
+    }
+  }
+------------------------------------------------------------------------------------
+#endif
 
 
 namespace ngsiv2
 {
+
 /* ****************************************************************************
 *
 * ForwardingInformation -
 */
-struct ForwardingInformation
+class ForwardingInformation
 {
-  int          lastFailure;  // FIXME: in Subscrition.h we use int for this, not long long as in lastNotification. why?
-  int          lastSuccess;  // FIXME: in Subscrition.h we use int for this, not long long as in lastNotification. why?
```

Comment:
No clue... maybe we didn't realize at the very begining and we haven't change for the fear of breaking backward compability at DB level? The change shouldn't hurt but I'm not fully sure...